README.md

```diff
@@ -1,33 +1,34 @@
 # <img src="./flexpa_logo.png" height="60px" align="center" alt="Flexpa logo"> Flexpa Quickstart
 
-This repository is a companion to Flexpa's [quickstart guide](https://www.flexpa.com/docs/guides/quickstart) which provides a detailed explanation of how this code example works.
+This repository is a companion to Flexpa's [quickstart guide](https://www.flexpa.com/docs/guides/quickstart) which also provides a detailed explanation of how this code example works.
 
 ## Prerequisites
 
-You must have valid publishable and secret keys for Flexpa API. [Generate](https://www.flexpa.com/docs/sdk/api#keys) test mode API keys or [contact us](https://automatemedical.typeform.com/to/mtwkkY2r?typeform-source=quickstart) to obtain production API keys. Once you have the API keys, you're ready to go!
+The quickstart assumes you have [NodeJS (v16.14.2+) and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) already installed.
 
-Node must be installed on your machine.
+## Clone repo
 
-## Clone the Repository
-
-First you must clone the quickstart repository. You can do this using HTTPS:
+First, clone the quickstart repository:
 
 ```bash
 git clone https://github.com/flexpa/quickstart.git
-cd quickstart
 ```
 
-## Update environment variables
+## Set API keys
 
-You'll need to add your Flexpa publishable and secret keys to a `.env` file in the client and server directories.
-Rename the files `.env.template` to `.env` in both directories and update the values to your publishable and secret keys.
+Second, create an account in our [developer portal](https://portal.flexpa.com/) to get your test mode keys. You can [contact us](https://flexpa.typeform.com/intake) later for live mode access.
```

Comment:
> You can [contact us](https://flexpa.typeform.com/intake) later for live mode access.

They actually can sign up in Portal and request Live Mode access there.

Reply:
Thanks for catching. Just pushed that tweak!